test/common/router/retry_state_impl_test.cc

```diff
@@ -167,6 +168,12 @@ class RouterRetryStateImplTest : public testing::Test {
   NiceMock<TestRetryPolicy> policy_;
   NiceMock<Upstream::MockClusterInfo> cluster_;
   TestVirtualCluster virtual_cluster_;
+  Stats::TestUtil::TestSymbolTable symbol_table_;
```

Comment:
Don't make a new symbol table; just declare stats_store_ first, and then use stats_store_.symbolTable().